amulet/level/interfaces/chunk/anvil/anvil_1934.py

```diff
@@ -1,17 +1,49 @@
 from __future__ import annotations
 
+from typing import List, Tuple, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from amulet.api.chunk import Chunk
+
+from amulet_nbt import CompoundTag, ByteTag
+
+from .base_anvil_interface import ChunkDataType, ChunkPathType
+
 from .anvil_1912 import Anvil1912Interface as ParentInterface
 
 
 class Anvil1934Interface(ParentInterface):
+    isLightOn: ChunkPathType = (
+        "region",
+        [("Level", CompoundTag), ("isLightOn", ByteTag)],
+        ByteTag,
+    )
     """
     Made lighting optional
+    Made game recalculate lighting 
     """
 
     def __init__(self):
         super().__init__()
         self._set_feature("light_optional", "true")
 
+        self._register_decoder(self._decode_islighton)
+        self._register_encoder(self._encode_islighton)
+
+    def _decode_islighton(
+        self, chunk: Chunk, data: ChunkDataType, floor_cy: int, height_cy: int
+    ):
+        if self.get_layer_obj(data, self.isLightOn) == ByteTag(1):
```

Comment:
only call this once. Store the value.

Reply:
This checks if the tag and value exists in the chunk and if the set value is 1, if it is it stores it, then format.py,
 ```py
changed_ |= chunk.misc.get("isLightOn", None) is not None  #true
```

```py
if chunk.misc.get("isLightOn", None):  # is true
            self.set_layer_obj(data, self.isLightOn, ByteTag(0) ) # sets it to 0 
```
this is according to my tests

Reply:
It looks like nothing gets committed to the originally loaded object data, even after a save it still returns true, until you reload the world in amulet. make it a little tedious when testing.

Reply:
The purpose of the decoder and encoder is to read the data and write it back exactly as it was found.
The lighting changes in the save operation are a separate step.